Code/Tools/AWSNativeSDKInit/include/AWSNativeSDKInit/AWSMemoryInterface.h

```diff
@@ -15,31 +15,12 @@
 #include <cstddef>
 #endif
 
-#include <AzCore/Memory/Memory.h>
 #include <AzCore/Memory/SystemAllocator.h>
+#include <AzCore/Memory/AllocatorWrappers.h>
 
 namespace AWSNativeSDKInit
 {
-    class AWSNativeSDKAllocator final
-        : public AZ::SystemAllocator
-    {
-    public:
-        AZ_CLASS_ALLOCATOR(AWSNativeSDKAllocator, AZ::SystemAllocator, 0);
-        AZ_TYPE_INFO(AWSNativeSDKAllocator, "{8B4DA42F-2507-4A5B-B13C-4B2A72BC161E}");
-
-        ///////////////////////////////////////////////////////////////////////////////////////////
-        // IAllocator
-        const char* GetName() const override
-        {
-            return "AWSNativeSDKAllocator";
-        }
-
-        const char* GetDescription() const override
-        {
-            return "Allocator used by the AWSNativeSDK";
-        }
-        ///////////////////////////////////////////////////////////////////////////////////////////
-    };
+    AZ_ALLOCATOR_DEFAULT_GLOBAL_WRAPPER(AWSNativeSDKAllocator, AZ::SystemAllocator, "{8B4DA42F-2507-4A5B-B13C-4B2A72BC161E}")
```

Comment:
We can actually delete the entire Code/Tools/AWSNativeSDKInit folder from the repo.

Reply:
Is there already a separate task for that work?

Reply:
No there is not.